engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -257,6 +257,22 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    Matrix4 CompLabelLocalTransform(
+        const Point3& position,
+        const Quat& rotation,
+        const Vector3& scale,
+        const Vector3& size,
+        uint32_t pivot)
+    {
+        // Move pivot to (0,0). Rotate around (0,0). Move pivot to position.
+        return dmTransform::ToMatrix4(
+            dmTransform::Mul(
+                dmTransform::Transform(Vector3(position), rotation, 1.0f),
+                dmTransform::Transform(CalcPivotDelta(pivot, mulPerElem(scale, size)), Quat::identity(), 1.0f)
```

Comment:
Note to self: Don't put "using namespace Vectormath::Aos;" at global scope!
We don't want things to unintentionally leak into other translation units.

Reply:
But this using is in cpp file, it shouldn't leak to other translation units... Or maybe I misunderstand something here?

Reply:
It might if one would do batch (or "unity") compiling to speed things up.
I.e. a "big.cpp", containing:
```
#include <a.cpp>
#include <b.cpp>
...
```

It's not an unreasonable optimization to do in the future.

Reply:
got it :) yeah, I might have heard of speeding things up this way in some gdc talk